Gemfile

```diff
@@ -32,49 +32,50 @@ gem "simple_captcha2", "0.4.3", require: "simple_captcha"
 
 # Background processing
 
-gem "sidekiq", "4.2.9"
+gem "sidekiq", "4.2.10"
 
 # Scheduled processing
 
-gem "sidekiq-cron", "0.4.5"
+gem "sidekiq-cron", "0.6.3"
 
 # Compression
 
-gem "uglifier", "3.1.2"
+gem "uglifier", "3.2.0"
 
 # Configuration
 
 gem "configurate", "0.3.1"
 
 # Cross-origin resource sharing
 
-gem "rack-cors", "0.4.1", require: "rack/cors"
+gem "rack-cors", "1.0.1", require: "rack/cors"
 
 # CSS
 
-gem "autoprefixer-rails",     "6.7.6"
+gem "autoprefixer-rails",     "7.1.2.3"
 gem "bootstrap-sass",         "3.3.7"
 gem "bootstrap-switch-rails", "3.3.3"
-gem "compass-rails",          "2.0.5"
+gem "compass-rails",          "3.0.2"
 gem "sass-rails",             "5.0.6"
+gem "sprockets",              "2.12.4"
 
 # Database
 
 group :mysql, optional: true do
-  gem "mysql2", "0.4.5"
+  gem "mysql2", "0.4.8"
 end
 group :postgresql, optional: true do
-  gem "pg",     "0.19.0"
+  gem "pg",     "0.21.0"
 end
 
 
-gem "activerecord-import", "0.17.1"
+gem "activerecord-import", "0.19.1"
 
 # File uploading
 
-gem "fog-aws",     "1.2.1"
-gem "carrierwave", "0.11.2"
-gem "mini_magick", "4.6.1"
+gem "carrierwave", "1.1.0"
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `carrierwave` should appear before `fog-aws`.